testinfra/modules/package.py

```diff
@@ -165,7 +165,16 @@ def version(self):
 class RpmPackage(Package):
     @property
     def is_installed(self):
-        return self.run_test("rpm -q %s", self.name).rc == 0
+        result = self.run("rpm -q %s", self.name)
+        if result.succeeded:
+            return True
+
+        if "not installed" in result.stdout:
```

Comment:
Doesn't rpm support many language depending on LANG/LC_ALL environment variables ? In which case this might not work.

Reply:
Good point. I will add an additional check here.